os/community-platforms.md

```diff
@@ -0,0 +1,47 @@
+# Community supported platforms
+
+CoreOS Container Linux supports a growing community of open source contributors. This page lists the most fully developed integrations for Container Linux on cloud and other platforms.
```

Comment:
it feels disingenuous to call these necessarily "open source contributors".

Many of them are using random closed source scripts to import our images.

"Fully developed" also is a little too much of a claim for quality I think.

My attempt to convey less "support" without it sounding negative.

The CoreOS Container Linux community has provided support for Container Linux on a number of platforms beyond those [officially supported](..) by CoreOS.

The platforms and providers listed below each provide support and documentation for running Container Linux:

Reply:
"officially supported" would simply link to the Container Linux landing page, and the "Cloud Providers" section.